packages/engine-test-utils/src/__tests__/common-all/sidebar.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { getSidebar, DefaultSidebar } from "@dendronhq/common-all";
+import { runEngineTestV5 } from "../../engine";
+import { ENGINE_HOOKS } from "../../presets";
+
+describe("GIVEN sidebar config input", () => {
```

Comment:
for snapshot tests, can you make sure to both add some comments for what we're looking for as well as some behavioral tests?  see [[dendron://dendron.docs/dev.process.qa.test.snapshot-testing]]

Reply:
[`4726ada` (#3448)](https://github.com/dendronhq/dendron/pull/3448/commits/4726adab1c530d4cdfbe9954167ed7d63793f705)